_gsocproposals/2019/proposal_AthenaMTResourceMonitoring.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Multi-thread safe resource monitoring infrastructure for the ATLAS experiment
+layout: gsoc_proposal
+project: ATLAS
+year: 2019
+organization:
+  - UCIrvine
+  - UOslo
```

Comment:
This name should match the one defined in the `organization` attribute in [`_gsocorgs/2019/uoslo`](https://github.com/HSF/hsf.github.io/pull/467/files#diff-855fddf114496dd43ff4bd1f476bccba). Otherwise the project will not appear on the list of project of this organization.

You can either replace this `UOslo` by `UiO` or modify the organization description ( [`_gsocorgs/2019/uoslo`](https://github.com/HSF/hsf.github.io/pull/467/files#diff-855fddf114496dd43ff4bd1f476bccba))

Reply:
Thanks for catching that. I updated the latter. Please let me know :)